Explorer/Assets/Scripts/Utility/Primitives/PrimitivesUtility.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Utility.Primitives
+{
+    public class PrimitivesUtility
+    {
+        public static Vector2[] FloatArrayToV2List(IList<float> uvs)
+        {
+            var uvsResult = new Vector2[uvs.Count / 2];
```

Comment:
We already pool `UVs` in the corresponding primitives' factories. It is incorrect to replace them with a newly allocated instance

Reply:
You can consider every `new()` for arrays banned